scripts/mirror.js

```diff
@@ -282,71 +244,55 @@ const bumpChromeAndroid = (originalData, sourceData) => {
 
-    if (sourceData.notes) {
-      newData.notes = updateNotes(
-        sourceData.notes,
-        /Internet Explorer/g,
-        'Edge',
-      );
+  if (chromeFalse) {
+    if (
+      typeof sourceData.version_removed === 'string' &&
+      compareVersions.compare(sourceData.version_removed, '79', '<=')
+    ) {
+      // If this feature was removed before Chrome 79, it's not present in Edge
+      return { version_added: false };
     }
-  } else if (source == 'chrome') {
-    newData = originalData == undefined ? sourceData : originalData;
-
-    let chromeFalse =
-      sourceData.version_added === false ||
-      sourceData.version_removed !== undefined;
-    let chromeNull = sourceData.version_added === null;
 
-    if (originalData === undefined) {
-      newData.version_added = chromeFalse ? false : chromeNull ? null : '≤79';
-    } else {
-      if (!chromeFalse && !chromeNull) {
-        if (originalData.version_added == true) {
-          newData.version_added = '≤18';
-        } else {
-          if (
-            sourceData.version_added == true ||
-            Number(sourceData.version_added) <= 79
-          ) {
-            if (originalData.version_added == false) {
-              newData.version_added = '79';
-            } else if (originalData.version_added == null) {
-              newData.version_added = '≤79';
-            }
-          } else {
-            newData.version_added = sourceData.version_added;
-          }
-        }
-      } else if (chromeFalse) {
-        if (originalData.version_added && !originalData.version_removed) {
-          newData.version_removed = '79';
-        }
+    if (
+      typeof sourceData.version_added === 'string' &&
+      compareVersions.compare(sourceData.version_added, '79', '<=')
+    ) {
+      // If the feature was added before Chrome 79 but removed afterwards
+      if (typeof originalData.version_added == 'string') {
+        newData.version_added = originalData.version_added;
+      } else {
+        newData.version_added = '79';
       }
     }
-
-    let newNotes = combineNotes(
-      updateNotes(sourceData.notes, /Chrome/g, 'Edge'),
-      originalData.notes,
-    );
-
-    if (newNotes) {
-      newData.notes = newNotes;
+    if (originalData.version_added && !originalData.version_removed) {
+      newData.version_removed = '79';
+    }
+  } else if (chromeNull) {
+    newData.version_added = null;
+  } else {
+    if (sourceData.version_added === true) {
+      newData.version_added =
+        originalData.version_added === true ? '≤18' : true;
+    } else if (compareVersions.compare(sourceData.version_added, '79', '<=')) {
+      newData.version_added =
+        originalData.version_added === null ? '≤79' : '79';
+    } else {
+      newData.version_added = sourceData.version_added;
     }
   }
 
+  let newNotes = updateNotes(sourceData.notes, /Chrome(?! ?OS)/g, 'Edge');
+  if (newNotes) newData.notes = newNotes;
```

Comment:
Maybe just preference, but I think the most common JS coding standards prefer blocks here:
```suggestion
  if (newNotes) {
    newData.notes = newNotes;
  }
```